test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -779,6 +790,18 @@ spec setup =
                 x.round . to_vector . should_equal [1, -2, 4]
                 x.truncate . to_vector . should_equal [1, -2, 3]
 
+    Test.group prefix+"Date truncation" <|
+        Test.specify "should be able to truncate a column of Date_Times" pending=pending_datetime <|
```

Comment:
nit: since the whole group is related to date-time, the pending status can be applied to the whole group
```suggestion
    Test.group prefix+"Date truncation" pending=pending_datetime <|
        Test.specify "should be able to truncate a column of Date_Times" <|
```

Reply:
Done